### ISSUE TITLE
Collaboration app

### ISSUE THREAD

**Issue by Nullf0x** [BruteDev/sode-os#2]
Hey guys, to discuss over matters related to the project, install the slack integration and connect to the brutedev channel.

**Comment by CreatorB** [comment]
actually, i have been on slack channel, maybe you can give us link brutedev channel or invite us by mail.

**Comment by Nullf0x** [comment]
Yep, one second, yeah, I will invite you in a minute. By the way, thanks!
